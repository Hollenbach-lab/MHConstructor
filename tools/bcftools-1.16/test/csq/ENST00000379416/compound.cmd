{bin}/bcftools csq -B 3 -f ENST00000379416.fa -g ENST00000379416.gff compound.del-snp.vcf       | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -B 3 -f ENST00000379416.fa -g ENST00000379416.gff compound.ins-del.vcf       | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -B 3 -f ENST00000379416.fa -g ENST00000379416.gff compound.ins-ins.vcf       | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -B 3 -f ENST00000379416.fa -g ENST00000379416.gff compound.ins-snp.vcf       | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
echo
{bin}/bcftools csq -B 3 -f ENST00000379416.fa -g ENST00000379416.gff single.vcf                 | ../sort-csq | {bin}/bcftools query -f'%POS\t%REF\t%ALT\t%BCSQ\n'
