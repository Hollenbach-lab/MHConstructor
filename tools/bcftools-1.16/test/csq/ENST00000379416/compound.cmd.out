53592	GC	G	@53595
53595	A	T	frameshift|XDH|ENST00000379416|protein_coding|-|216LLR..1334>216I*|53592GC>G+53595A>T

53594	A	AGC	@53595
53595	ACT	A	missense&inframe_altering|XDH|ENST00000379416|protein_coding|-|215EL>215VL|53594A>AGC+53595ACT>A

53594	A	AGC	@53595
53595	A	ACTC	frameshift|XDH|ENST00000379416|protein_coding|-|216LLR..1334>216ECC*|53594A>AGC+53595A>ACTC

53594	A	AGC	@53595
53595	A	T	frameshift|XDH|ENST00000379416|protein_coding|-|216LLR..1334>216SC*|53594A>AGC+53595A>T

53594	AA	AGCT	frameshift|XDH|ENST00000379416|protein_coding|-|216LLR..1334>216SC*|53594AA>AGCT
